composer.json

```diff
@@ -4,7 +4,7 @@
     "type": "neos-package",
     "license": "MIT",
     "require": {
-        "neos/content-repository": "^3.0 || ^4.0"
+        "neos/content-repository": "^3.0 || ^4.0 || ^5.0"
```

Comment:
Did you check if `PsrSystemLoggerInterface` is available in 3, 4 and version 5 ?

Reply:
Ah you're right of course.
I've removed 3.0 now from the dependencies.

The PsrSystemLoggerInterface was introduced in flow 5.0, so CR 4.0 is fine.